src/components/TheNavigation.vue

```diff
@@ -6,23 +6,36 @@
     <div class="navlinks">
       <router-link to="/about-us" tag="button">About Us</router-link>
       <router-link to="/events" tag="button">Events</router-link>
-      <router-link to="/login" tag="button">Log In</router-link>
-      <button v-on:click="logout">Log Out</button>
+      <access-control :roles="['guest']">
```

Comment:
Okay, so this all makes sense. But we should try and keep user concepts and enums consistent across the frontend and the backend. Here's what the backend uses consistently (how consistently @liammoyn ?)

(note: GP and PF should be expanded to longer names that are recognizable. If it doesn't read like english it doesn't work) 

```
public enum PrivilegeLevel {
  GP(0), PF(1), ADMIN(2);

  private int val;

  PrivilegeLevel(int val) {
    this.val = val;
  }

  public int getVal() {
    return val;
  }

  public static PrivilegeLevel from(Integer val) {
    for (PrivilegeLevel privilegeLevel : PrivilegeLevel.values()) {
      if (privilegeLevel.val == val) {
        return privilegeLevel;
      }
    }
    throw new IllegalArgumentException(String.format("Given num (%d) that doesn't correspond to any PrivilegeLevel", val));
  }
}
```

Reply:
dont have brain capacity to read this, slack me if you need input